src/ObjectsConfig.js

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Bad naming
Perhaps JavascriptHelper (if you have a better idea...)
Create static methods:
getDomParserObject and getWebSocketObject